server/website/script/fixture_generators/knob_settings/oracle/create_knob_settings.py

```diff
@@ -0,0 +1,218 @@
+        fields['tunable'] = True
+        fields['minval'] = 0
+        fields['maxval'] = 3000000000  # 3G
+        fields['default'] = 0
+    if fields['name'].upper() == 'OPEN_CURSORS':
+        fields['tunable'] = False
+        fields['minval'] = 200
+        fields['maxval'] = 400
+        fields['default'] = 300
+    if fields['name'].upper() == 'DB_FILE_MULTIBLOCK_READ_COUNT':
+        fields['tunable'] = False
+        fields['minval'] = 64
+        fields['maxval'] = 256
+        fields['default'] = 128
+    if fields['name'].upper() == 'optimizer_index_cost_adj'.upper():
+        fields['tunable'] = False
+        fields['minval'] = 1
+        fields['maxval'] = 10000
+        fields['default'] = 100
+    if fields['name'].upper() == 'OPTIMIZER_USE_PENDING_STATISTICS':
+        fields['tunable'] = False
+        fields['minval'] = None
+        fields['maxval'] = None
+        fields['default'] = False
+    if fields['name'].upper() == 'OPTIMIZER_USE_INVISIBLE_INDEXES':
+        fields['tunable'] = False
+        fields['minval'] = None
+        fields['maxval'] = None
+        fields['default'] = False
+    if fields['name'].upper() == 'OPTIMIZER_USE_SQL_PLAN_BASELINES':
+        fields['tunable'] = False
+        fields['minval'] = None
+        fields['maxval'] = None
+        fields['default'] = True
+    if fields['name'].upper() == 'OPTIMIZER_CAPTURE_SQL_PLAN_BASELINES':
+        fields['tunable'] = False
+        fields['minval'] = None
+        fields['maxval'] = None
+        fields['default'] = False
+    if fields['name'].upper() == 'DISK_ASYNCH_IO':
+        fields['tunable'] = True
+        fields['vartype'] = 5
+        fields['enumvals'] = 'TRUE,FALSE'
+        fields['default'] = 'TRUE'
+
+
+def main():
+    final_metrics = []
+    with open('oracle.txt', 'r') as f:
+        num = 0
+
+        lines = f.readlines()
+        for line in lines:
+            line = line.replace("\n", "")
+            pattern = re.compile(r"^\s+|\s+$")
+            line = re.sub(pattern, '', line)
+            if len(line) <= 0:
+                continue
+            if line == 'DESCRIPTION' or line == 'NAME'\
+                    or line == 'TYPE' or line[0] == '-':
+                continue
+            if num == 0:
+                entry = {}
+                entry['model'] = 'website.KnobCatalog'
+                fields = {}
+                fields['name'] = line
+                num = 1
+                continue
+            if num == 1:
+                if line == '3' or line == '6':
+                    fields['vartype'] = 2
+                    fields['default'] = 0
+                elif line == '1':
+                    fields['vartype'] = 4
+                    fields['default'] = False
+                else:
+                    fields['vartype'] = 1
+                    fields['default'] = ''
+                num = 2
+                continue
+            if num == 2:
+                num = 0
```

Comment:
Hum... you are cycling `num`, right? 0, 1, 2, 0, 1, 2, ... If so, then why not insert  `num = (num + 1) % 3` after line 211 (same identation level of those `ifs`)?